src/docs_website/assets/style/style.css

```diff
@@ -708,6 +708,7 @@ article {
 
         code {
           color: var(--gray-12);
+          word-break: break-all;
```

Comment:
Whoa, I missed that we use CSS nesting (actually, I missed that css nesting exists!)! Is it a sufficiently supported feature? I guess we are aiming at mostly tech-literate people with modern browsers, so we should be fine!

Reply:
Works in all major browsers since Dec '23. https://caniuse.com/css-nesting